src/components/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,111 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import {isBrowser} from 'utils/executionEnvironment';
+
+import styles from './index.scss';
+
+export default class DropDown extends Component {
+  static propTypes = {
+    children: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.element),
+      PropTypes.element
+    ]).isRequired,
+    direction: PropTypes.oneOf(['up', 'down']),
+    pullRight: PropTypes.bool,
+    showOnHover: PropTypes.bool,
+    title: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.element
```

Comment:
It accepts element because of this situation: https://github.com/kununu/app-admin/pull/26/files#r246248270